### PR TITLE
Forms Editor Update

### PR DESCRIPTION
**Pull request by eshanrnh**
1. Updated the articles structure within the Editor section for V9.
2. Updated images wherever applicable.

### CODE REVIEW

File: Add-ons/UmbracoForms/Editor/Attaching-Workflows/Workflow-Types/index.md [hunk 1]
```diff
@@ -1,126 +1,121 @@
...
 
-## Send email
+    ![Send email](images/send-email.png)
 
-![Send email](images/send-email.png)
+    Sends the result of the Form to the specified email address. The following configuration can be set:
 
-Send the result of the form to a specified email address.
+  - Email (required)
+  - SenderEmail - also configurable in `Config/umbracosettings.config`
```
Comment (jonat123) [inline Add-ons/UmbracoForms/Editor/Attaching-Workflows/Workflow-Types/index.md:65]: I don't think this is the same in V9 🤔

As we are not using the .Config files no more, i think we will need to hear one of the forms people how this is done in V9.

Comment (eshanrnh) [reply Add-ons/UmbracoForms/Editor/Attaching-Workflows/Workflow-Types/index.md:65]: oh yes, Good catch.. the UI still has this path though.. 
image

Comment (jonat123) [reply Add-ons/UmbracoForms/Editor/Attaching-Workflows/Workflow-Types/index.md:65]: Yeah i noticed that as well... let's reach out to Some of the people working with forms, they should be able to give us an answer :)

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Date/index.md [hunk 2]
```diff
@@ -1,25 +1,27 @@
...
 
-This includes the Moment.js library to help with the date locale formatting & the appropriate changes to Pikaday to support the locales. If you wish to use a different DatePicker component this is the file that you would customize to your needs.
+The **DatePicker.cshtml** includes the `moment-with-locales.min.js` library to help with the date locale formatting and the appropriate changes to Pikaday to support the locales. If you wish to use a different DatePicker component, edit the **DatePicker.cshtml** file as per your needs.
 
 ## Configure the Year range
+
 The Date picker has a configuration setting to control the number of years shown in the picker. The default value is 10 years.
 
-You can find the settings for changing this configuration in `~/App_Data/UmbracoForms/umbracoforms.config`:
+You can configure the settings in the `UmbracoForms.config` file located in `~/App_Plugins/UmbracoForms/`:
```
Comment (jonat123) [inline Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Date/index.md:24]: For V9 it is in the AppSetting file as seen here: https://our.umbraco.com/Documentation/Add-ons/UmbracoForms/Developer/Configuration/index-v9#date-picker-field-type-configuration :)

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Recaptcha2/index.md [hunk 3]
```diff
@@ -1,18 +1,22 @@
 ---
 versionFrom: 7.0.0
+versionTo: 9.0.0
 ---
 
 # reCAPTCHA V2
 
-![reCAPTCHA v2](images/recaptcha2.png)
-
 You need to configure your site keys adding your public and private keys in the `UmbracoForms.config` file located in `~/App_Plugins/UmbracoForms/`:
```
Comment (jonat123) [inline Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Recaptcha2/index.md:8]: It is the same with ReChapta: https://our.umbraco.com/Documentation/Add-ons/UmbracoForms/Developer/Configuration/index-v9#recaptcha-v2-field-type-configuration :)

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Recaptcha3/index.md [hunk 4]
```diff
@@ -4,11 +4,18 @@ versionFrom: 8.0.0
 
 # reCAPTCHA V3
 
-You need to configure your site keys adding your public and private keys in the `UmbracoForms.config` file located in `~/App_Plugins/UmbracoForms/`:
+You need to configure your site keys by adding your public and private keys in the `UmbracoForms.config` file located in `~/App_Plugins/UmbracoForms/`:
```
Comment (jonat123) [inline Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Recaptcha3/index.md:7]: Same with Rechapta V3 it needs to be set in the AppSetting file :)

**Comment by jonat123** [comment]
@eshanrnh Looks really good! :D The only thing is the config parts, i have made a comment where i saw that it mentioned the V8 config, and we need to also add how it can be done in V9, So i also linked to the Config documentations for forms and V9 :)

**Comment by eshanrnh** [comment]
Thanks, Jonathan 😊 I have made the changes to the file names and added code samples where required.

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Date/index.md [hunk 5]
```diff
@@ -1,28 +1,36 @@
...
+To achieve localized date, a Razor partial view is included at `/Views/Partials/Forms/Themes/default/DatePicker.cshtml`.
 
-This includes the Moment.js library to help with the date locale formatting & the appropriate changes to Pikaday to support the locales. If you wish to use a different DatePicker component this is the file that you would customize to your needs.
+The **DatePicker.cshtml** includes the `moment-with-locales.min.js` library to help with the date locale formatting and the appropriate changes to Pikaday to support the locales. If you wish to use a different DatePicker component, edit the **DatePicker.cshtml** file as per your needs.
 
 ## Configure the Year range
+
 The Date picker has a configuration setting to control the number of years shown in the picker. The default value is 10 years.
 
-You can find the settings for changing this configuration in `~/App_Data/UmbracoForms/umbracoforms.config`:
```
Comment (jonat123) [inline Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Date/index.md:22]: If this article is still valid for V8 as well as V9, then we should still say that they can configure it from the UmbracoForm.config :) 

So you could add the sentence back and add: _You can find the settings for changing this configuration for Umbraco 8 in `~/App_Data/UmbracoForms/umbracoforms.config`:_

And then also re-add the configuration for V8 as well.

I hope it makes sense 😊

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Recaptcha3/index.md [hunk 6]
```diff
@@ -4,11 +4,21 @@ versionFrom: 8.0.0
 
 # reCAPTCHA V3
 
-You need to configure your site keys adding your public and private keys in the `UmbracoForms.config` file located in `~/App_Plugins/UmbracoForms/`:
+You need to configure your site keys by adding your public and private keys in the `appSettings.json` file:
 
-```xml
-<setting key="RecaptchaV3SiteKey" value="..." />
-<setting key="RecaptchaV3PrivateKey" value="..." />
```
Comment (jonat123) [inline Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Recaptcha3/index.md:11]: The same here, unless we make a specific article for V9, we need to still have the configuration for V8 in the article as well :)

**Comment by jonat123** [comment]
@eshanrnh great! I have made a few comments 🙂

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Date/index.md (hunk 5 above)
Comment (eshanrnh) [reply Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Date/index.md:22]: It definitely makes sense. Not sure what was I thinking while taking it off.. 😐 Added the content back in 😊

File: Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Recaptcha3/index.md (hunk 6 above)
Comment (eshanrnh) [reply Add-ons/UmbracoForms/Editor/Creating-a-Form/Fieldtypes/Recaptcha3/index.md]: Done

**Pull request merged**